<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Calibration_function_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>校准功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Superposition_function_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>叠加功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Noise_reduction_function_button_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>降噪功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Alignment_function_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>对齐功能</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="featuresWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>521</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>校准</string>
     </attribute>
     <widget class="QStackedWidget" name="Calibration_widget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>481</width>
        <height>191</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QPushButton" name="Calibration_button">
        <property name="geometry">
         <rect>
          <x>338</x>
          <y>154</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>一键校准</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Flat_field_button">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>154</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>平场校准</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Dark_field_button">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>154</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>暗场校准</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Bias_field_button">
        <property name="geometry">
         <rect>
          <x>229</x>
          <y>154</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>偏置场校准</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save_file_button_calibration">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>125</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Select_file_button_calibration">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Object_file_button_flat">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>67</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>平场文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Object_file_button_dark">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>38</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>暗场文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Object_file_button_bias">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>96</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>偏置场文件</string>
        </property>
       </widget>
       <widget class="QLabel" name="Object_file_label_dark">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>38</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择暗场文件路径</string>
        </property>
       </widget>
       <widget class="QLabel" name="Save_file_label_calibration">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>125</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件路径</string>
        </property>
       </widget>
       <widget class="QLabel" name="Select_file_label_calibration">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>9</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件路径</string>
        </property>
       </widget>
       <widget class="QLabel" name="Select_file_label_flat">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>67</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择平场文件路径</string>
        </property>
       </widget>
       <widget class="QLabel" name="Object_file_label_bias">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>96</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择偏置场文件路径</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>对齐</string>
     </attribute>
     <widget class="QStackedWidget" name="Alignment_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="Object_file_label_aligning">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>100</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择目标文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Select_file_button_aligning">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save_file_button_aligning">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>130</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件</string>
        </property>
       </widget>
       <widget class="QLabel" name="Select_file_label_aligning">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Object_file_button_aligning">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>70</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>对齐文件</string>
        </property>
       </widget>
       <widget class="QLabel" name="Save_file_label_aligning">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>160</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Alignment_button">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>190</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>对齐</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>降噪</string>
     </attribute>
     <widget class="QStackedWidget" name="Noise_reduction_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>171</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="Select_file_label_noise_reduction">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Select_file_button_noise_reduction">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Noise_reduction_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>高斯降噪</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save_file_button_noise_reduction">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>70</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件</string>
        </property>
       </widget>
       <widget class="QLabel" name="Save_file_label_noise_reduction">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>100</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Bilateral_reduction_button">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>双边降噪</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>叠加</string>
     </attribute>
     <widget class="QStackedWidget" name="Superposition_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="Save_file_label_superposing">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>100</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Select_file_button_superposing">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Superposition_button">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>130</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>叠加</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save_file_button_superposing">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>70</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件</string>
        </property>
       </widget>
       <widget class="QLabel" name="Select_file_label_superposing">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件路径</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
     </widget>
    </widget>
    <widget class="QWidget" name="Baryer_tab">
     <attribute name="title">
      <string>解拜尔</string>
     </attribute>
     <widget class="QStackedWidget" name="Baryer_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_15">
       <widget class="QLabel" name="Save_file_label_baryer">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>100</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Select_file_button_baryer">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Superposition_baryer">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>130</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>解拜尔</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save_file_button_baryer">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>70</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存文件</string>
        </property>
       </widget>
       <widget class="QLabel" name="Select_file_label_baryer">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>300</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件路径</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_16"/>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Baryer_function_button_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>解拜尔</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Select_file_button_aligning</tabstop>
  <tabstop>Object_file_button_aligning</tabstop>
  <tabstop>Save_file_button_aligning</tabstop>
  <tabstop>Alignment_button</tabstop>
  <tabstop>Superposition_function_button</tabstop>
  <tabstop>Noise_reduction_function_button_2</tabstop>
  <tabstop>Calibration_function_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
